### ISSUE TITLE
Error in sending images

### ISSUE THREAD

**Issue by cesagonchu** [mambax7/adslight#5]
When I send an image for an ad, on the user side, it does not appear, there is a transparent thumbnail instead.
When I edit the ad, it tells me that there is no image.
I don't get any error message.

I then tried to send an image, on the admin side.
I then get this error, when I click on the image thumbnail:
`Error: Class 'XoopsModules/Adslight/Pictures' not found in file /modules/adslight/class/PicturesHandler.php line 213`

**Comment by mambax7** [comment]
Thanks for reporting it.
I'll look into it this weekend

**Comment by cesagonchu** [comment]
Two users are waiting to update Adslight on their website. Did you make any progress?

**Comment by mambax7** [comment]
Should finish this weekend, as the latest.

**Comment by cesagonchu** [comment]
Following your changes, I found a new error:
https://github.com/mambax7/adslight/issues/6#issue-1025567754

We are making progress, that's the main thing. 👍🏻

**Comment by cesagonchu** [comment]
No. After checking "yes" to "Do you want to add Photos Now" and clicking the submit button, it takes me back to the website home page.

**Comment by cesagonchu** [comment]
Okay, I see what the problem is.
When I add an image for an ad by clicking on the green button, it works.
/modules/adslight/add.php

When I add an image by selecting a category, that's when it doesn't work.
/modules/adslight/viewcats.php?cid=25
/modules/adslight/addlisting.php?cid=25

**Comment by mambax7** [comment]
Thanks for narrowing it down. I'll work on this use case as well. 
Unfortunately, the code is messy, as there were several different developers rewriting parts of it over several years, so there are different styles and approaches mixed in, and sometimes it's hard to follow the logic. But we'll get there :)
